app/components/project_submissions/item_component.html.erb

```diff
@@ -8,7 +8,10 @@
 
     <div class="flex flex-col md:flex-row md:items-center">
       <%= link_to 'View code', project_submission.repo_url, target: '_blank', rel: 'noreferrer', class: 'button button--gray font-semibold md:mr-4', data: { test_id: 'view-code-btn' } %>
-      <%= link_to 'Live preview', project_submission.live_preview_url, target: '_blank', rel: 'noreferrer', class: 'button button--gray font-semibold mt-5 md:mt-0 md:mr-4', data: { test_id: 'live-preview-btn' } %>
+
+      <% if project_submission.lesson.has_live_preview? %>
```

Comment:
Mega nit: Demeter

A delegated `previewable?` or similar method might be a nice minor addition to the project submission model? I was going to suggest doing it in the component, but we're calling the same logic from the form too

Reply:
Nice! `previewable ` is a _much_ better name. What do you think about us renaming the db column to that? we'd get those nice predicate methods for free.

Reply:
Yeah mint even better, love that :heart:

Reply:
I created a follow up issue to rename it https://github.com/TheOdinProject/theodinproject/issues/4003

Best to do it after the react version gets the 🪓